powerauth-webflow-authentication-mtoken/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/mtoken/controller/MobileAppApiController.java

```diff
@@ -142,12 +142,16 @@ protected AuthMethod getAuthMethodName() {
             }
 
             // Get the list of operations for given user
-            final List<GetOperationDetailResponse> operationList = getOperationListForUser(userId);
-            if (operationList == null) {
-                // Next step operation failed, return empty operation list
+            List<GetOperationDetailResponse> operationList;
+            Map<String, GetOperationConfigResponse> operationConfigs;
+            try {
+                operationList = getOperationListForUser(userId);
+                operationConfigs = getOperationConfigs(operationList);
+            } catch (AuthStepException e) {
+                // Next step operation list failed, return empty operation list
```

Comment:
Should we add logging here?

Reply:
This is intentional, the Exception would be logged twice.

Reply:
ok